### DISCUSSION TITLE
Currently, I use a separate node to represent "or" but I wonder if there's a better visual to do this

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1491] [Help]
![Different way to do an or node](https://github.com/freeplane/freeplane/assets/147225942/7e840690-e2fc-4711-93ee-7b0fa368f108)

So as the title and picture demonstrate, I am currently using that setup to represent this phrase: "A plaintiff must have performed or must have been willing and able to perform"

And because these nodes are a part of requirements for a rule, I can't just add another sister node that has the "or" option.

![sfsifsd](https://github.com/freeplane/freeplane/assets/147225942/64485d24-90ff-4c28-bc96-8cc496d76617)

I'm wondering if there's a better technique with Freeplane to represent "or" than just making another node that contains the word

**Comment by euu2021** [comment]
This kind of situation, I write like this:
![image](https://github.com/freeplane/freeplane/assets/77707706/503b38e0-2984-4681-bd69-a0cfbfb6732d)

So, I put the ANDs and ORs are before each element, because it's clear enough to me. 

To make it more clean, one idea may be to color code AND and OR elements. For example, AND elements as orange, and OR elements as blue:
![image](https://github.com/freeplane/freeplane/assets/77707706/61f5d039-2b99-4d6c-b185-549ab056887d)

With this color coding, the logic gets visually intuitive, for me.

**Reply by adamfreeplaneuser01** [reply]
<img width="802" alt="This works great" src="https://github.com/freeplane/freeplane/assets/147225942/5d3c4ae2-d25b-4680-9a2c-e96b8d476c38">
This works great, I think!

**Reply by adamfreeplaneuser01** [reply]
The only thing is, is there any way to display some sort of "key" at the top of the screen that always you to state what each color represents

**Reply by euu2021** [reply]
> The only thing is, is there any way to display some sort of "key" at the top of the screen that always you to state what each color represents

I can't think of any easy way to create an overlay in FP. I would probably leave the key opened in a second tab, name the tab "key" (yes, we can name tabs, now 🎉), and check it when necessary.

**Reply by EdoFro** [reply]
I agree. This is the way

**Comment by sifran-github** [comment]
I never use freeplane for these types of structures, but i would do it this way:
![image](https://github.com/freeplane/freeplane/assets/5730355/4750d339-705e-4848-93d8-cc82567b5795)

maybe with different colors to represent the and/or operation:
![image](https://github.com/freeplane/freeplane/assets/5730355/d9ec1aad-58b9-4645-aa7d-5f82dc7ecd17)
